src/Components/Notifications/NotificationsList.tsx

```diff
@@ -278,36 +290,38 @@ export default function NotificationsList({
   const unsubscribe = () => {
     navigator.serviceWorker.ready
       .then(function (reg) {
-        setIsSubscribing(true);
-        reg.pushManager
-          .getSubscription()
-          .then(function (subscription) {
-            subscription
-              ?.unsubscribe()
-              .then(async function (_successful) {
-                const data = {
-                  pf_endpoint: "",
-                  pf_p256dh: "",
-                  pf_auth: "",
-                };
+        if (reg.pushManager !== null && reg.pushManager !== undefined) {
```

Comment:
same here too, let's negate the logic and return (similar to the previous comment), that makes it more readable